code/game/jobs/job/security.dm

```diff
@@ -42,6 +43,7 @@
 		ACCESS_SCIENCE_LVL2,
 		ACCESS_SCIENCE_LVL3,
 		ACCESS_SCIENCE_LVL4,
```

Comment:
Lower the science access here.

Reply:
Science access is used for SCP-containment-cell access; the containment engineer wouldn't be able to do his job without this.